qa/tasks/cephfs/test_journal_migration.py

```diff
@@ -78,7 +78,7 @@ def test_journal_migration(self):
                               "--path", "/tmp/journal.json"], 0)
         p = self.fs.tool_remote.run(
             args=[
-                "python",
+                "python3",
```

Comment:
I'm not sure about this change. Isn't nautilus still tested with python2?

Reply:
This has caused failures as expected,
http://pulpito.ceph.com/yuriw-2020-05-01_18:26:19-fs-wip-yuri2-testing-2020-05-01-1612-nautilus-distro-basic-smithi/5006837/

Reply:
This change is not required, the reason of failure from the log above is that the script is python2 but python is python3, so there is a syntax error. Instead this should be rewritten so the `print` uses parenthesis, or as follows:
```
event_count_str = self.fs.tool_remote.sh('''
python -c "import json; print(len(json.load(open('/tmp/journal.json'))))"
'''.strip()
event_count = int(event_count_str)
```

Reply:
I only manually backport the part of 7e87f80a8754c93c10ff937844ad456d4a236f57 for this backport pr.
That should be better for this backport pr.
The suggestions of @kshtsk say(use remote.sh to avoid extra StringIO) I think that should be another backport pr?
This backport pr should only for py3 in cephfs qa bugs.